build_local.sh

```diff
@@ -45,4 +45,4 @@ python3.5 -m venv /tmp/dcos_build_venv
 ./prep_local
 
 # Build a release of DC/OS
-release create `whoami` local_build $tree_variants
```

Comment:
Does `--noop` flag mean that release create will be skipped for local build? Couldn't get the intention behind this change.

Reply:
Good question. The `--noop` flag has existed for a very long time, it appears to skip the step where cloudformation templates and installer artifacts are uploaded to S3. It's certainly a bad name but I don't want to make breaking changes. If you like I can make `--local` imply `--noop` - that said, I prefer having command-line options be orthogonal.